supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -1,10 +1,879 @@
+
+*Use it*: yes
+
+*Incorrect forms*: Any form using uppercase letters or omitting the hyphen
+
+*See also*: xref:core-management-subsystem[Core Management Subsystem]
+
+// ***********************
+// Terms starting with 'D'
+// ***********************
+
+[discrete]
+[[datasource]]
+==== datasource (noun)
+*Description*: The "datasource" subsystem is used to create and configure data sources and to manage JDBC database drivers. Write in lowercase as one word. See xref:datasource-subsystem[Datasource Subsystem] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: Datasource, data source
+
+*See also*: xref:datasource-subsystem[Datasource Subsystem]
+
+[discrete]
+[[datasource-subsystem]]
+==== Datasource Subsystem (noun)
+*Description*: Use "Datasource Subsystem" when referring to the xref:datasource[datasource] subsystem in titles and headings. Capitalize each word. See xref:datasource[datasource] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: Datasource subsystem, datasource subsystem
+
+*See also*: xref:datasource[datasource]
+
+[discrete]
+[[deployment-scanners-heading]]
+==== Deployment Scanners (noun)
+*Description*: Use "Deployment Scanners" when referring to the xref:deployment-scanner[deployment-scanner] subsystem in titles and headings. Capitalize each word. See xref:deployment-scanner[deployment-scanner] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: Deployment scanners, deployment scanners
+
+*See also*: xref:deployment-scanner[deployment-scanner]
+
+[discrete]
+[[deployment-scanner]]
+==== deployment-scanner (noun)
+*Description*: The "deployment-scanner" subsystem is used to configure scanners to check for applications to deploy. Write in lowercase as two words separated by a hyphen. See xref:deployment-scanners-heading[Deployment Scanners] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: Any form using uppercase letters or omitting the hyphen
+
+*See also*: xref:deployment-scanners-heading[Deployment Scanners]
+
+[discrete]
+[[domain-mode]]
+==== domain mode (noun)
+*Description*: Do not use "domain mode" to refer to the running instance of JBoss EAP server. See xref:managed-domain[managed domain] entry for the correct usage.
+
+*Use it*: no
+
+*Incorrect forms*:
+
+*See also*: xref:managed-domain[managed domain]
+
+// ***********************
+// Terms starting with 'E'
+// ***********************
+
+[discrete]
+[[ee]]
+==== ee (noun)
+*Description*: The "ee" subsystem is used to configure functionality in the Jakarta Enterprise Edition platform. Write in lowercase as one word. See xref:ee-heading[EE] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: EE, Ee
+
+*See also*: xref:ee-heading[EE]
+
+[discrete]
+[[ee-heading]]
+==== EE (noun)
```

Comment:
Okay, so I commented earlier about the whole "Subsystem" maybe not needing to be capitalized, but I did actually like it in the term. Because all of these subsystem entries look like they're just regular words that can be used. For some reason, the first few entries had subsystem after them, but then that stops.

I'd recommend adding "subsystem" after each of the subsystem entries (both for the normal and the title/heading entries), so that it's clearer that it's a subsystem. I assume that "subsystem" is pretty much required afterward anyways.

Reply:
Fixed